invenio_app_ils/records/resolver/jsonresolver/document_item.py

```diff
@@ -32,16 +34,36 @@ def items_resolver(document_pid):
                 "medium": item.get("medium"),
                 "status": item.get("status"),
                 "description": item.get("description"),
+                "shelf": item.get("shelf"),
+                "internal_location": {
+                    "location": {
+                        "name": item.get("internal_location")
```

Comment:
if you use `get`, I would do `item.get("internal_location", {})` otherwise it will fail, WDYT?